index.js

```diff
@@ -10,12 +10,12 @@ module.exports = {
         module.router = makeRouter(clientConstructor);
         const normalizedPath = path.join(__dirname, 'test', 'functional');
         fs.readdirSync(normalizedPath)
-        .map(function(file) {
-            require('./test/functional/' + file);
+        .map((file) => {
+            return require(`./test/functional/${file}`);
```

Comment:
In cases like this when all the function body does is returning a single value it's nice to write it as

```
(file => require('./test/functional/' + file))
```